openapi/preprocess_spec.py

```diff
@@ -415,6 +422,21 @@ def inline_primitive_models(spec, excluded_primitives):
     for k in to_remove_models:
         del spec['definitions'][k]
 
+def remove_bad_descriptions_recursively(model, pattern):
```

Comment:
please add
```python
if pattern is None:
  return
```

to the top to exit early and for clarity.

Reply:
oops, this is the recursive function, I see the check in the function below, please ignore this.